k8s/agones/schema.yaml

```diff
@@ -0,0 +1,128 @@
+applicationApiVersion: v1beta1
+
+properties:
+  APP_INSTANCE_NAME:
+    type: string
+    x-google-marketplace:
+      type: NAME
+
+  NAMESPACE:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+
+  INSTALL_XONOTIC_DEMO:
+    type: boolean
+    title: Register Xonotic fleet (example opensource game) into the cluster.
+    default: true
+
+  marketplace_integration.service_account:
+    type: string
+    title: Service account used to bootstrap Agones in the cluster.
```

Comment:
This is super nit, but please don't put dot at the end of the title.

```suggestion
    title: Service account used to bootstrap Agones in the cluster
```

Reply:
Done